src/app/file-manager/reducers/downloadables.reducer.ts

```diff
@@ -11,6 +11,26 @@ export interface DownloadableState {
     items: Item[];
 }
 
+export class DownloadableStateImpl implements DownloadableState {
```

Comment:
This is OK but I don't see why you didn't change `DownloadableState` to be a concrete class instead of adding an Impl class. As far as I can tell that would work fine and would avoid adding another class type to the code.